nucypher/crypto/ferveo/dkg.py

```diff
@@ -93,7 +94,7 @@ def derive_decryption_share(
     try:
         derive_share = _VARIANTS[variant]
     except KeyError:
-        raise Exception(f"invalid variant {variant}")
+        raise ValueError(f"Invalid variant {variant}")
```

Comment:
Just a thought: I've noticed that there is a similar error message being thrown by `ferveo` Python bindings. I don't think it's that confusing, but if someone we're to search for `Invalid variant` in `nucypher` this is where they would end up, not in `ferveo` or `nucypher` core.